packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -8537,11 +8537,21 @@ export enum SemanticColorSlots {
     // (undocumented)
     bodyBackground = 0,
     // (undocumented)
-    bodyText = 1,
+    bodyDivider = 4,
     // (undocumented)
-    disabledBackground = 2,
+    bodyStandoutBackground = 1,
     // (undocumented)
-    disabledText = 3,

```

Comment:
We should re-add the removed items and just add to this list instead.

Reply:
done.